Core/Sources/Fundamental/Models/ChatModel.swift

```diff
@@ -23,6 +23,7 @@ public struct ChatModel: Codable, Equatable, Identifiable {
         case openAICompatible
         case googleAI
         case ollama
+        case anthropic
```

Comment:
Do you think you can rename it or set the string value to `claude` so that it matches CopilotForXcode and users will be able to use the existing models imported from CopilotForXcode?

I think anthropic would be a better name but claude was already used in CopilotForXcode. You can keep the `antropic` in other places.

Reply:
I renamed all to Claude to keep consistency with CopilotForXcode app 😉